goltsmannn2/src/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include "hello_world.cpp"
```

Comment:
Do not include the source file here. This just copy pastes the source file here. You should use cmake to build a linkable library and then just include the header and not the source implementation as this is still just a giga binary.